management_api_app/models/schemas/workspace.py

```diff
@@ -88,3 +93,14 @@ class Config:
                 "workspaceId": "49a7445c-aae6-41ec-a539-30dfa90ab1ae",
             }
         }
+
+
+class WorkspacePatch(BaseModel):
```

Comment:
Is this class itself going to get extended when we add new fields which can be patched? If not then we could rename this WorkspaceEnabledPatch. It could read better and if we add new patch classes existing tests will not need changing everytime we add new fields.

Reply:
@sachinkundu My plan was for it to be extended. This is the schema for the general patch.

If you want to have different WorkspaceEnabledPatch and WorkspaceDescriptionPatch, you would need to have different PATCH endpoints, is that what you envision? i.e. that we should instead have PATCH /api/workspace/{workspace_id}/enabled? or can you explain a bit more how you envision this working?

Reply:
Ah right. That's a bummer. I would like to have a single endpoint for patch but a more nuanced payload control. Is it possible to have a Union type? That is payload is of type WorkspacePatchEnabled or WorkspacePatchDisplayname

If not then go ahead.

Reply:
Looks like you can union - so I have changed it to WorkspacePatchEnabled for now and will union once there is a need for it